rma_sale/models/stock_move.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    is_sale_delivery = fields.Boolean(
+        compute='_compute_is_sale_delivery',
+        store=True,
```

Comment:
Why do you need this field and make it computed? This will overhead the installation of the module. I think it's better to not be stored unless it impacts in performance, which I don't think if you are using it in single records.